lib/commands/createcache.js

```diff
@@ -45,15 +58,9 @@ function createCache(opts, request, done){
         "compression" : {
           "minimumSizeInKB" : 512
         },
-        "description" : "Store Response",
+        "description" : opts.description,
         "diskSizeInMB" : 1024,
-        "distributed" : true,
-        "expirySettings" : {
-          "expiryDate" : {
-            "value" : "12-31-9999"
-          },
-          "valuesNull" : false
-        },
```

Comment:
We should continue to use the default values that were there before, but with your changes they would override any of the defaults if/when provided.

Reply:
PR is updated to use default values for expiry